cypress/integration/dryRunner/cardLog.js

```diff
@@ -0,0 +1,47 @@
+import s from './selectors'
+const DECK_ID =
+  'MU4xLDFOMiwxSTIsMU4zLDFONCwxTjUsMU42LDFONjIsMU42MywxTjgsMU4xMCwxTjEz'
+const HAND = ['N1', 'N3', 'I2', 'N10']
+
+describe('Dry-runner — Card Log', () => {
+  beforeEach(() => {
+    cy.visit(`/deck/${DECK_ID}/dry-run?mode=MANUAL`).drDrawHand(HAND)
+  })
+
+  it('should log a card when played', () => {
+    cy.drPlay('N1')
+      .get(s.CARD_LOG)
+      .get('[class="DryRunnerCardLog__image"]')
```

Comment:
```suggestion
      .get('.DryRunnerCardLog__image')
```

… or better, we add a `data-testid`.